flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/operations/DataStreamQueryOperation.java

```diff
@@ -31,6 +31,8 @@
 /**
  * Describes a relational operation that reads from a {@link DataStream}.
  *
+ * <p>This is only used for testing.
```

Comment:
this `DataStreamQueryOperation` is only used in `TableTestBase` for adding test DataStream with statistic. and it will not used for user interface, such as `StreamTableEnvironmentImpl#fromDataStream`.